src/components/image.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC } from "react"
+import classnames from "classnames"
+
+interface Props {
+  source: string
+  onLoad?: () => void
+  alt?: string
+  objectFit?: "cover" | "contain"
+  galleryImage?: boolean
+  width?: number
+  height?: number
+  onError?: (e) => void
+}
+
+const Image: FC<Props> = ({
+  source,
+  objectFit,
+  onLoad,
+  alt,
+  width,
+  height,
+  onError,
+  galleryImage,
```

Comment:
can we find a name that imposes less how the image is used and more on what the effect of the prop will be? is it `fullSize` or `fullScreen` or something?

Reply:
I went for `fullScreenHeight` but `fullScreen` might be enough actually 👍